volmdlr/wires.py

```diff
@@ -1125,6 +1125,21 @@ def __init__(self, primitives: List[volmdlr.core.Primitive2D],
     #         equal = (equal and prim1 == prim2)
     #     return equal
 
+    def __eq__(self, other_):
+        if other_.__class__.__name__ != self.__class__.__name__:
+            return False
+        if len(self.primitives) != len(other_.primitives):
+            return False
+        equal = 0
+        for prim1 in self.primitives:
+            for prim2 in other_.primitives:
+                if (prim1 == prim2 or prim1.reverse() == prim2
+                        or prim2.reverse() == prim1 or prim1.reverse() == prim2.reverse()):
+                    equal += 1
+        if equal == len(self.primitives) and equal == len(other_.primitives):
+            return True
+        return False
```

Comment:
Just a small optimization.
```suggestion
    def __eq__(self, other_):
        if other_.__class__.__name__ != self.__class__.__name__:
            return False
        if len(self.primitives) != len(other_.primitives):
            return False
        equal = 0
        for prim1 in self.primitives:
            reverse1 = prim1.reverse()
            found = False
            for prim2 in other_.primitives:
                reverse2 = prim2.reverse()
                if (prim1 == prim2 or reverse1 == prim2
                        or reverse2 == prim1 or reverse1 == reverse2):
                    equal += 1
                    found = True
            if not found:
                return False
        if equal == len(self.primitives):
            return True
        return False
```

Reply:
Thanks !